src/puppetlabs/puppetdb/config.clj

```diff
@@ -289,23 +288,27 @@
     (cond
      (nil? vardir)
      (throw (IllegalArgumentException.
-             "Required setting 'vardir' is not specified. Please set it to a writable directory."))
+             (format "%s %s"
+                     (trs "Required setting ''vardir'' is not specified.")
+                     (trs "Please set it to a writable directory."))))
```

Comment:
I might prefer str or str join if we need to break things up like this, but it's not critical, so I'm fine with a merge either way.